azure-functions/acbs-function/mappings/facility/facility-covenant-amend.test.js

```diff
@@ -1,49 +1,148 @@
 const facilityCovenantAmend = require('./facility-covenant-amend');
 
 describe('facilityCovenantAmend', () => {
-  describe('when there is an amendment amount', () => {
-    const amendment = { amount: 123.456 };
+  describe('when the facility amount is being amended', () => {
+    it('should return amount with two decimal points', () => {
+      // Arrange
+      const payload = { amount: 123.456 };
+      const expected = {
+        targetAmount: 123.46,
+      };
 
-    it('returns the target amount rounded to 2 decimal places', () => {
-      const result = facilityCovenantAmend(amendment);
-      expect(result).toEqual({ targetAmount: 123.46 });
+      // Act
+      const result = facilityCovenantAmend(payload);
+
+      // Assert
+      expect(result).toEqual(expected);
+    });
+
+    it('should returns the amount as a number with two decimal points when the amount is a string', () => {
```

Comment:
not a blocker, can be done in a later PR
```suggestion
    it('should return the amount as a number with two decimal points when the amount is a string', () => {
```